core/Azure.Mcp.Core/src/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -109,22 +106,21 @@ private static void EnableAzureMonitor(this IServiceCollection services)
 
                 r.AddService("azmcp", version)
                     .AddTelemetrySdk();
-            })
-            .UseAzureMonitorExporter(options =>
-            {
-#if DEBUG
-                options.EnableLiveMetrics = true;
-                options.Diagnostics.IsLoggingEnabled = true;
-                options.Diagnostics.IsLoggingContentEnabled = true;
```

Comment:
Do we still need these for troubleshooting in debug mode?

Reply:
Now that Azure Monitor Exporter is release-mode only I don't think this is necessary anymore. Unless there's something I'm missing and we should have a way to enable these more detailed captures for release-ready builds.